test/extended/prometheus/prometheus_builds.go

```diff
@@ -61,12 +53,12 @@ var _ = g.Describe("[sig-instrumentation][sig-builds][Feature:Builds] Prometheus
 			// allow for some retry, a la prometheus.go and its initial hitting of the metrics endpoint after
 			// instantiating prometheus tempalte
 			var err error
-			for i := 0; i < maxPrometheusQueryAttempts; i++ {
+			for i := 0; i < helper.MaxPrometheusQueryAttempts; i++ {
```

Comment:
nit: By making this 403 polling a `helper` function, you could make `maxPrometheusQueryAttempts` and `prometheusQueryRetrySleep` internal to the `helper` package.  Not a big deal, but I think that would make for a slightly cleaner API.